docs/vsphere-golden-image.md

```diff
@@ -95,7 +95,18 @@ In case no firewall rule exist, you must create it by running the following Powe
     New-NetFirewallRule -DisplayName 'OpenSSH Server (sshd)' -LocalPort 22 -Enabled True -Direction Inbound -Protocol TCP -Action Allow 
 ```
 
-## 4. Set up incoming connection for container logs
+## 4. Enable Containers feature on the Windows OS
+
+Explicitly enable the `Containers` feature on the Windows OS. This feature is not turned on by default on 
+Windows Server 2022.
+
+```powershell
+ Install-WindowsFeature -Name Containers
+```
+
+Note that a system reboot is required after toggling this feature.
```

Comment:
Why? the `Containers` feature is not used during the golden image creation, i.e. Step 5 does not depend on it.